components/core/src/Stopwatch.hpp

```diff
@@ -1,29 +1,44 @@
 #ifndef STOPWATCH_HPP
 #define STOPWATCH_HPP
+// #define PROFILING
```

Comment:
The changes to Stopwatch largely seem to be duplicating what the Profiler class is for. One of the advantages of the Profiler class is that the code can be easily disabled at compile time, so we can leave the calls within the code.